webui/src/lib/components/repository/changes.jsx

```diff
@@ -109,7 +114,17 @@ export const ChangeEntryRow = ({ entry, showActions, onRevert }) => {
                     {diffIndicator}
                 </td>
                 <td className="tree-path">
-                    <span>{pathText}</span>
+                    {(entry.path_type === "common_prefix") ? (
```

Comment:
funny JSX indentation here at the end of ` {...}` block.

Reply:
Fixed